### PR TITLE
Auto recommendation

### PR DESCRIPTION
**Pull request by mindedj**
refactored recommend time and date into functions
recommends date or time on "wrong" date
can recommend time on specific date and the user can accept it
[tesztek.txt](https://github.com/szegedai/hun-appointment-chatbot/files/6101354/tesztek.txt)

### CODE REVIEW

File: actions/actions.py [hunk 1]
```diff
@@ -95,6 +95,34 @@ def is_multiple_days(candidates):
     else:
         return False
 
+def rec_date(appointments):
```
Comment (nsoma97) [inline actions/actions.py:98]: Could you clean these two functions up a bit? Make them follow similar logic, only one return statement at the end, etc..

**Review by nsoma97 — APPROVED** [review]
Overall nicely done, I couldn't find a malfunction when testing it.

File: domain.yml [hunk 2]
```diff
@@ -9,16 +13,18 @@ intents:
...
+- dates
+- times
 slots:
   date:
     type: text
   time:
-    type: text
-
-
+    type: rasa.shared.core.slots.TextSlot
```
[the remark this answers is not in the document]
Comment (mindedj) [reply domain.yml:22]: There's no difference, just when I start RasaX it automatically changes domain.yml to these values (+e2e and form) and I seem to have forgotten to change it back. I can revert these or leave them as they don't change funcionality but RasaX would change them back. (not sure if we'll use rasax though)

[the remark this answers is not in the document]
Comment (nsoma97) [reply domain.yml:22]: Hmm, it's weird that `date` remains text but `time` doesn't. We can leave it as it is in this case, I was just curious if it changes any functionality. :+1:

**Comment by nsoma97** [comment]
I've played around with it, couldn't seem to produce any issue and all my comments are resolved, so I think you can go ahead and merge this. :+1:

File: domain.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (mindedj) [reply domain.yml:22]: Double checked it, I messed up. It indeed changes both, I only reverted date, not all. I will push the corrected (all in new format) code in a minute.

**Pull request merged**